content/_index.md

```diff
@@ -13,3 +13,4 @@ To learn more about OSM:
 * Go through all the [getting started articles](/docs/getting_started/) to install OSM and run a sample application.
 * Read an [overview of OSM](/docs/overview/about/) and more about [the design of its components](/docs/overview/osm_components/).
 * Run additional example scenarios from the [Demos](/docs/demos/) section.
+* Consider using the [OSM add-on](https://docs.microsoft.com/azure/aks/open-service-mesh-about) with an [Azure Kubernetes Service cluster](https://docs.microsoft.com/azure/aks).
```

Comment:
This seems out of place in the "learn more about OSM" section.

It makes more sense to add this in a section dedicated to managed offerings.

Reply:
@zr-msft based on our discussion offline, it might be more appropriate to add an `adopters.md` file in the [osm](https://github.com/openservicemesh/osm) github repo and leave this section as is. I don't think we need a dedicated page on managed solutions, if any, on the website.